coverage_comment/coverage.py

```diff
@@ -138,6 +151,26 @@ def generate_coverage_markdown(coverage_path: pathlib.Path) -> str:
     )
 
 
+def _make_coverage_info(data: dict) -> CoverageInfo:
+    """Build a CoverageInfo object from a "summary" or "totals" key."""
+    return CoverageInfo(
+        covered_lines=data["covered_lines"],
+        num_statements=data["num_statements"],
+        percent_covered=compute_coverage(
+            num_covered=data["covered_lines"],
+            num_total=data["num_statements"],
+            num_branches_covered=data.get("covered_branches", 0),
+            num_branches_total=data.get("num_branches", 0),
+        ),
+        missing_lines=data["missing_lines"],
+        excluded_lines=data["excluded_lines"],
+        num_branches=data.get("num_branches"),
+        num_partial_branches=data.get("num_partial_branches"),
```

Comment:
And I'm now realizing I should've defaulted these to `0` as well.

```suggestion
        num_branches=data.get("num_branches", 0),
        num_partial_branches=data.get("num_partial_branches", 0),
```

Reply:
If you've realized this now, then I'd say this deserves a unit test :)